framework/PageForm/Inputs/PageFormMarkdown.tsx

```diff
@@ -153,8 +154,10 @@ export function PageFormMarkdown<
               </DropZone>
             )}
             {isExpanded && preview && (
-              <PreviewContainer className="pf-v5-c-form-control pf-m-disabled">
-                <TextContent>{<ReactMarkdown>{value}</ReactMarkdown>}</TextContent>
+              <PreviewContainer>
+                <TextContent>
+                  {<ReactMarkdown remarkPlugins={[remarkGfm]}>{value}</ReactMarkdown>}
```

Comment:
We already have the `MarkdownEditor` component (https://github.com/ansible/ansible-ui/blob/main/frontend/hub/common/MarkdownEditor.tsx) but it may need to be slightly modified.

Also, why is the `remark-gfm` package needed?

Reply:
@jamestalton , could you weigh in on this?

Reply:
remark-gfm is used for tables in markdown. 
react-markdown does not support that without that plugin.